### PR TITLE
feat: jwt filtering and get current authorized user id in user controller

### PR DESCRIPTION
**Pull request by hea9549**

### CODE REVIEW

File: src/main/java/orangetaxiteam/cocoman/config/JwtTokenProvider.java [hunk 1]
```diff
@@ -25,8 +24,7 @@
     @Value("spring.jwt.secret")
     private String secretKey;
 
-    private long tokenValidMilisecond = 1000L * 60 * 60 * 24 * 7; // 1 week
-    private final UserDetailsService userDetailsService;
+    private long tokenValidMillisecond = 1000L * 60 * 60 * 24 * 7; // 1 week
```
Comment (hihiboss) [inline src/main/java/orangetaxiteam/cocoman/config/JwtTokenProvider.java]: 요거 상수로 해주세요

File: src/main/java/orangetaxiteam/cocoman/web/UserController.java [hunk 2]
```diff
@@ -33,20 +34,22 @@ public UserDTO signUp(@RequestBody UserCreateRequestDTO userCreateRequestDTO) {
...
+            @AuthenticationPrincipal String currentUserId
+    ) {
+        if (!id.equals(currentUserId)) throw new AccessDeniedException(String.format("no permission to id [%s] ", id));
         return this.userApplicationService.findById(id);
     }
 
     @PutMapping(value = "/{id}")
     @ResponseStatus(value = HttpStatus.OK)
-    public UserDTO updateUser(@PathVariable String id, @RequestBody UserUpdateRequestDTO userUpdateRequestDTO) {
+    public UserDTO updateUser(@PathVariable String id,
```
Comment (hihiboss) [inline src/main/java/orangetaxiteam/cocoman/web/UserController.java]:
```suggestion
    public UserDTO updateUser(
    	@PathVariable String id,
```

**Comment by hea9549** [comment]
@hihiboss 다했다요~

**Review by GunHee5719 — APPROVED** [review]
LGTM

**Review by hihiboss — APPROVED** [review]

**Pull request merged**